apps/src/templates/codeDocs/MethodWithOverloads.jsx

```diff
@@ -29,6 +30,7 @@ export function SingleMethod({
           <ParametersTable
             parameters={method.parameters}
             programmingEnvironmentLanguage={programmingEnvironmentLanguage}
+            isSmallWindow={isSmallWindow}
```

Comment:
Just a thought - would it be easier to use [Context](https://reactjs.org/docs/context.html) here rather than plumbing this value through multiple components?

Reply:
Looking at the docs it says Context is best when multiple components need to read the data, which isn't the case here. I think it might be overkill for this case and make the components harder to reuse